pkg/cvo/updatepayload.go

```diff
@@ -189,13 +190,40 @@ func (r *payloadRetriever) fetchUpdatePayloadToDir(ctx context.Context, dir stri
 		},
 	}
 
+	// Prune older jobs while gracefully ignoring errors.
+	r.pruneJobs()
```

Comment:
I think this call site should be the one ignoring the errors rather than ignoring it in the function itself.

Reply:
Is that not what it is already doing? It returns errors in the function, but doesn't even bother with them when called here.

Reply:
https://github.com/openshift/cluster-version-operator/pull/186/files/53e5b922c64ac7f534f5df84dcaada10b364db22#diff-d540a41404f678a1c438f4c1e5b92a87R221 is not returning any errors, but warning?

^^ should be `errors.Wrapf(err, "failed to delete job %q", job.Name)`

also we might be in better shape if we try to delete all jobs and aggregate those errors before returning.

Reply:
@abhinavdahiya Done, PTAL.